packages/create-block-app/create-block-app.js

```diff
@@ -157,7 +157,18 @@ const usage = commandLineUsage(helpSections);
     await fs.writeJson(blockMetadataPath, blockMetadata, { spaces: 2 });
   }
 
+  const chalk = (await import("chalk")).default;
+
+  console.log(
+    chalk.bold("-".repeat(Math.min(process.stdout.columns ?? 48, 48))),
+  );
   console.log(
-    `Your ${blockName} block is ready to code in ${resolvedBlockPath}.\nRun 'yarn install && yarn dev' to get started`,
+    `Your ${chalk.bold(blockName)} block is ready to code in ${chalk.bold(
+      resolvedBlockPath,
+    )}.\n` +
+      `Run ${chalk.blue(`cd ${resolvedBlockPath}`)} and then \n` +
+      `${chalk.blue("yarn install && yarn dev")} or ${chalk.blue(
+        "npm install && npm run dev",
```

Comment:
Optional: if we mention `npm install` here, we could do the same [in docs](https://blockprotocol.org/docs/developing-blocks#creating-a-block).

Reply:
Thanks - done in https://github.com/blockprotocol/blockprotocol/pull/603/commits/cd1b786aa46d7cd98341c96b12de6a9d4ace44d8